README.md

```diff
@@ -1,16 +1,29 @@
-# Charm for MongoDB on Kubernetes
+# Charmed MongoDB on Kubernetes
+## Overview
 
-## Description
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production ready data platform on top of cloud native technologies.
 
-Fully supported solution for production-grade MongoDB on Kubernetes from Canonical.
+MongoDB is a popular NoSQL database application. It stores its data with JSON-like documents creating a flexible experience for users; with easy to use data aggregation for data analytics. It is a distributed database, so vertical and horizontal scaling come naturally.
```

Comment:
Again, you could link to MongoDB documentation here

Reply:
Since we already have one link to MongoDB I will refrain here